CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+## [1.61](https://github.com/brave/brave-ios/releases/tag/v1.61)
+
+### Web3
+
+- Implemented new UI for the “Transaction Details” page. ([#8379](https://github.com/brave/brave-ios/issues/8379))
+- Implemented new UI for the “Activity” tab in Brave Wallet. ([#8349](https://github.com/brave/brave-ios/issues/8349))
+- Updated UI and functionality for the "NFT" details page. ([#8064](https://github.com/brave/brave-ios/issues/8064))
+- Updated UI and functionality for “NFT” tab under “Portfolio.” ([#8063](https://github.com/brave/brave-ios/issues/8063))
+- Updated SNS interstitial page copy for new provider. ([#8515](https://github.com/brave/brave-ios/issues/8515))
+- Fixed NFT auto-discover and remove modals to be dismissed when Brave Wallet gets auto-locked. ([#8412](https://github.com/brave/brave-ios/issues/8412))
+
+### General 
+
+- Added share icon to the URL bar. ([#8173](https://github.com/brave/brave-ios/issues/8173))
+- Added separator between inactive tabs. ([#8361](https://github.com/brave/brave-ios/issues/8361))
+- Added  “Redeem Code” button for custom offer codes in the VPN subscription page. ([#8365](https://github.com/brave/brave-ios/issues/8353))
+- Added support for handling promotional in-app purchases. ([#8001](https://github.com/brave/brave-ios/issues/8001))
+- Added “Open Browser History” and “Open Tabs From Other Devices” options in “Siri Shortcuts” settings page. ([#8333](https://github.com/brave/brave-ios/issues/8333))
+- Added ability to suppress “Allow link to switch apps?” pop-ups as reported on HackerOne by amit5533. ([#7835](https://github.com/brave/brave-ios/issues/7835))
+- Added new keyboard shortcuts for tab navigation and page zoom on iPad. ([#7592](https://github.com/brave/brave-ios/issues/7592))
+- Fixed color scheme for Brave icons when using Dark Mode for lock screen and news widgets. ([#8502](https://github.com/brave/brave-ios/issues/8502))
+- Fixed links being clickable when using Slide-Over Mode on iPad. ([#8489](https://github.com/brave/brave-ios/issues/8489))
+- Fixed orientation lock for Brave sync screens. ([#6275](https://github.com/brave/brave-ios/issues/6275))
+- Fixed adding correct video to Playlist on https://twitter.com/. ([#8479](https://github.com/brave/brave-ios/issues/8479))
```

Comment:
Can probably word this a bit better. Maybe something along the lines of the following?

```
- Fixed incorrect video being added to Playlist on https://twitter.com/. ([#8479](https://github.com/brave/brave-ios/issues/8479))
```